rust/kernel/net/netlink.rs

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Netlink helpers.
+
+use crate::bindings;
+use crate::types::SavedAsPointer;
+
+/// Octets in one ethernet addr
```

Comment:
For things that are prose etc. avoid abbreviations and use the right capitalization for names. For instance, here `addr` does not refer to a parameter or something like that, so it should be "address" instead. Similarly, `ethernet` is not the name of e.g. a module, but it is the actual Ethernet (which is capitalized as "Ethernet").

```suggestion
/// Octets in one Ethernet address.
```